.buildkite/autogenerate_pipeline.py

```diff
@@ -61,6 +61,7 @@
 # This represents the version of the rust-vmm-container used
 # for running the tests.
 CONTAINER_VERSION = "v35"
+CONTAINER_VERSION_RISCV = CONTAINER_VERSION + "-riscv"
```

Comment:
Why we need `-riscv` in the tag?

Reply:
In `docker-publish.yml` of rust-vmm/rust-vmm-ci#159, since the `riscv64` image is actually a `x86_64` image with `qemu-system-riscv64` installed, I think it is not as standard enough as the other two platforms (they are tested on actual hardware) to have a name like `dev:riscv64`, that would be kind of misleading, people would tend to think that's a `riscv64` image. So I suggest we use name like `v35-riscv` for now, and standardize it afterwords

Reply:
LGTM, maybe a comment there will be helpful

Reply:
Noted, I will provide comments to them in next update